index.html

```diff
@@ -38,6 +38,10 @@
 
 	<link rel="icon" type="image/png" sizes="32x32" href="img/logo-32x32.png">
 	<link rel="icon" type="image/png" sizes="16x16" href="img/logo-16x16.png">
+	<!-- Tailwind -->
+	<script src="https://cdn.tailwindcss.com"></script>
```

Comment:
I would prefer to not introduce any new dependencies. We can achieve this footer using just CSS like the rest of the site.

Reply:
Yes, @MarcusOtter  we can acheive it by using normal CSS but by using Tailwind CSS it would be more easy.
And , in future we can add more features like switching to dark mode and light mode by using this dependency and I don't think adding a new dependency will effect the site and also your one of the issue fixing mobile navbar will be done by this dependency more efficiently as it had many pre defined features for it.

But if you want then, I'll again recreate this using CSS.

Reply:
I'd prefer raw CSS for now :)